2018/CNAME

```diff
@@ -0,0 +1 @@
+hackkean.org
```

Comment:
if I could get you to move this back to the root of the repo I could go ahead and get this merged. This file is the only file that needs to stay at the root of the repository, it tells github that this domain is okay to copy the github pages.